platform/solr/solr-dependencies/pom.xml

```diff
@@ -214,6 +214,17 @@
                             javax.xml.transform.stream,
                             javax.xml.xpath,
 
+                            <!--jetty-->
+                            org.eclipse.jetty.client;resolution:=optional,
+                            org.eclipse.jetty.client.api;resolution:=optional,
+                            org.eclipse.jetty.client.http;resolution:=optional,
+                            org.eclipse.jetty.client.util;resolution:=optional,
+                            org.eclipse.jetty.http;resolution:=optional,
+                            org.eclipse.jetty.http2.client;resolution:=optional,
+                            org.eclipse.jetty.http2.client.http;resolution:=optional,
+                            org.eclipse.jetty.util;resolution:=optional,
+                            org.eclipse.jetty.util.ssl;resolution:=optional,
```

Comment:
Making these imports optional. The Solr client seems to work fine for the most part without them. We only ran into an error when we tried to configure it to use basic auth. It's a little hard to say what is and isn't required by Solrj as it marks nothing as optional dependencies. The http2 packages are definitely optional, though

Bnd will automatically append the supported version range to these imports. So these will get: 
```
	org.eclipse.jetty.client;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.client.api;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.client.http;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.client.util;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.http;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.http2.client;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.http2.client.http;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.util;resolution:=optional;version="[9.4,10)",
	org.eclipse.jetty.util.ssl;resolution:=optional;version="[9.4,10)",
```